### PR TITLE
base bild

### PR DESCRIPTION
**Pull request by oksanatytanych**
- [DEMO LINK](https://oksanatytanich.github.io/layout_catalog/)
- [TEST REPORT LINK](https://oksanatytanich.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Comment by oksanatytanych** [comment]
- [DEMO LINK](https://oksanatytanych.github.io/layout_catalog/)
- [TEST REPORT LINK](https://oksanatytanych.github.io/layout_catalog/report/html_report/)

**Review by Andrey-Ivanov999 — APPROVED** [review]
I Think thats good:)

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,377 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <a class="header__logo" href="#home">
```
Comment (ivan-kadykalo) [inline src/index.html:15]: Checlist

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,377 @@
...
+
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a 
+          class="card__buy-link" 
+          href="#"
+          data-qa="card-hover"
+        >
+        Buy
```
Comment (ivan-kadykalo) [inline src/index.html:116]: Keep indentation

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**